lib/rspec/core/example.rb

```diff
@@ -138,7 +155,9 @@ def initialize(example_group_class, description, user_metadata, example_block=ni
         @example_block       = example_block
 
         @metadata = Metadata::ExampleHash.create(
-          @example_group_class.metadata, user_metadata, description, example_block
+          @example_group_class.metadata, user_metadata,
```

Comment:
> I always think multiple comma separated args on the same line is a smell, if one of them changes it'll churn the line with both of them making it less obvious in the diff what happened

So every time you call a method with multiple arguments you break the arguments up into separate lines?  I certainly haven't noticed that in your PRs....

In general, I don't think it's a code smell to put multiple args on the same line.  We do it _all over the place_ in RSpec and in every other ruby code base I've seen.

Regardless, while I like splitting a long arg list across a few lines to avoid a super long line, I dislike going as far as one-arg-per-line because of the extra vertical space it takes up.  That in turn limits how much code I can look at on my screen at one time.  I like the 3 line form I've used here because it avoids the single super long line, looks pretty decent visually (IMO) and avoids begin overly vertically long.



> Note the trailing comma, which means that if we add another argument, that line won't churn.

I like doing that, but 1.8.7 can't parse it so we can't adopt it for RSpec until we drop 1.8.7 support :(.